cfgov/cfgov/urls.py

```diff
@@ -23,7 +23,7 @@
     # url(r'^search/$', 'search.views.search', name='search'),
 
     url(r'^$', SheerTemplateView.as_view(), name='home'),
-    url(r'^home/(?P<path>.*)$', RedirectView.as_view(url='/%(path)s')),
+    url(r'^home/(?P<path>.*)$', RedirectView.as_view(url='/%(path)s', permanent=True)),
 
```

Comment:
its `permanent=True` by default. Django 1.9 changes it to `False` . Explicitly set to suppress upgrade warning